sources/packages/web/src/views/aest/institution/ApplicationOfferingChangeRequestForm.vue

```diff
@@ -0,0 +1,165 @@
+import { AESTRoutesConst } from "@/constants/routes/RouteConstants";
+import ApplicationOfferingChangeDetailsHeader from "@/components/aest/institution/ApplicationOfferingChangeDetailsHeader.vue";
+import ApplicationOfferingChangeRequestBanner from "@/components/aest/institution/ApplicationOfferingChangeRequestBanner.vue";
+import { ApplicationOfferingChangeRequestService } from "@/services/ApplicationOfferingChangeRequestService";
+import StudentApplicationOfferingChangeDetails from "@/components/aest/StudentApplicationOfferingChangeDetails.vue";
+import OfferingForm from "@/components/common/OfferingForm.vue";
+
+export default defineComponent({
+  components: {
+    ApplicationOfferingChangeRequestBanner,
+    StudentApplicationOfferingChangeDetails,
+    ApplicationOfferingChangeDetailsHeader,
+    OfferingForm,
+  },
+  props: {
+    studentId: {
+      type: Number,
+      required: true,
+    },
+    applicationId: {
+      type: Number,
+      required: true,
+    },
+    applicationOfferingChangeRequestId: {
+      type: Number,
+      required: true,
+    },
+  },
+
+  setup(props) {
+    const tab = ref("requested-change");
+    const headerDetails = ref({} as ApplicationOfferingChangeRequestHeader);
+    const applicationOfferingChangeRequestDetails = ref(
+      {} as ApplicationOfferingDetailsReviewAPIOutDTO,
+    );
+    onMounted(async () => {
+      applicationOfferingChangeRequestDetails.value =
+        await ApplicationOfferingChangeRequestService.shared.getApplicationOfferingDetailsForReview(
+          props.applicationOfferingChangeRequestId,
+        );
+      headerDetails.value = {
+        submittedDate:
+          applicationOfferingChangeRequestDetails.value.submittedDate?.toString(),
+        institutionName:
+          applicationOfferingChangeRequestDetails.value.institutionName,
+        locationName:
+          applicationOfferingChangeRequestDetails.value.locationName,
+        assessedBy: applicationOfferingChangeRequestDetails.value.assessedBy,
+        assessedDate:
+          applicationOfferingChangeRequestDetails.value.assessedDate?.toString(),
+        institutionId:
+          applicationOfferingChangeRequestDetails.value.institutionId,
+        status: applicationOfferingChangeRequestDetails.value.status,
+        updatedDate:
+          applicationOfferingChangeRequestDetails.value.updatedDate?.toString(),
+      };
+    });
+    const goBackRouteParams = computed(
+      () =>
+        ({
+          name: AESTRoutesConst.ASSESSMENTS_SUMMARY,
+          params: {
+            applicationId: props.applicationId,
+            studentId: props.studentId,
+          },
+        } as RouteLocationRaw),
+    );
+    return {
+      headerDetails,
+      AESTRoutesConst,
+      tab,
+      ApplicationOfferingChangeRequestStatus,
+      goBackRouteParams,
+      applicationOfferingChangeRequestDetails,
+      ApplicationOfferingChangeDetailsHeader,
+      ApplicationOfferingChangeRequestBanner,
```

Comment:
Is there a reason for the components to be returned here?

Reply:
removed it.